src/HOG.py

```diff
@@ -1,27 +1,27 @@
-# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring,exec-used,too-many-arguments,too-few-public-methods,no-self-use,too-many-locals,line-too-long,redefined-builtin,redefined-outer-name,bare-except
 
 from __future__ import print_function
 
-from evaluate import evaluate_class
-from DB import Database
-
-from skimage.feature import hog
-from skimage import color
+import os
 
-from six.moves import cPickle
 import numpy as np
 import scipy.misc
-import os
+from six.moves import cPickle
+from skimage import color
+from skimage.feature import hog
+
+from src.DB import Database
```

Comment:
```suggestion
from DB import Database
```

Reply:
if you do that, `DB` will be unable to import

Reply:
Ok

Reply:
It threw this error
```
$ python3 src/resnet.py
Traceback (most recent call last):
  File "src/resnet.py", line 14, in <module>
    from src.DB import Database
ModuleNotFoundError: No module named 'src'
```

Reply:
well, i have made py files under `src` submodules of `src`, thus we have to call them via `src.module`, outside `src` directory